packages/compiler-core/src/parse.ts

```diff
@@ -259,7 +259,9 @@ function parseChildren(
       const node = nodes[i]
       if (node.type === NodeTypes.TEXT) {
         if (!context.inPre) {
-          if (!/[^\t\r\n\f ]/.test(node.content)) {
+          const isAllWhitespacesText =
+            nodes.length === 1 && /^ +$/.test(node.content) // #7789
+          if (!/[^\t\r\n\f ]/.test(node.content) && !isAllWhitespacesText) {
```

Comment:
I was hoping to review this, but `parse.ts` was removed as part of the parser rewrite.

I think this section of the new parser may be the equivalent functionality:

https://github.com/vuejs/core/blob/cdb1d1795d21591659e2560e201ee4fbf1ea4aca/packages/compiler-core/src/parser.ts#L821-L831

I also confirmed that the original issue is still present in 3.4.27.

Reply:
Already updated, I really appreciate your time.